### PR TITLE
Add test for BaseActivity

### PR DESCRIPTION
**Pull request by tomrozb**

### CODE REVIEW

**Comment by tomrozb** [comment]
Unfortunately coverage reports are not generated: https://code.google.com/p/android/issues/detail?id=170607

File: .travis.yml [hunk 1]
```diff
@@ -8,9 +8,15 @@ android:
   - extra-google-m2repository
   - extra-android-m2repository
   - addon-google_apis-google-19
+  - sys-img-armeabi-v7a-android-19
+env:
+  - ANDROID_TARGET=android-19 ANDROID_ABI=armeabi-v7a
```
Comment (arturdm) [inline .travis.yml]: shouldn't we check with the minSdkVersion?

Comment (tomrozb) [inline .travis.yml]: that's a good idea

**Comment by arturdm** [comment]
+1

**Pull request merged**